ppx/ast_attributes.ml

```diff
@@ -130,7 +140,9 @@ let process_attributes_rev attrs : attr_kind * attribute list =
 let process_pexp_fun_attributes_rev attrs =
   List.fold_left
     ~f:(fun (st, acc) ({ attr_name = { txt; _ }; _ } as attr) ->
-      match txt with "mel.open" -> (true, acc) | _ -> (st, attr :: acc))
+      match txt with
+      | "mel.open" | "bs.open" -> (true, acc)
```

Comment:
What does this attribute do? It's not documented in melange.re.

Also, should `warn_if_bs_or_non_namespaced` be called in this case too?

Reply:
`@mel.open` extracts an OCaml `exn` from an external JS data source. Check some old docs here: https://melange.re/melange/Manual.html#__code_bs_open_code_type_safe_external_data_source_handling_since_1_7_0

Reply:
https://github.com/melange-re/melange-re.github.io/issues/157